app_vdi.py

```diff
@@ -0,0 +1,212 @@
+                    title='Hover description',
+                    children=[
+                        dcc.Upload(
+                            id='load-data',
+                            className='app__upload',
+                            children=html.Div([
+                                'Drag and Drop or ',
+                                html.A('Select Files')
+                            ]),
+                            multiple=False,
+                            style={
+                                'width': '100%',
+                                'height': '60px',
+                                'lineHeight': '60px',
+                                'borderWidth': '1px',
+                                'borderStyle': 'dashed',
+                                'borderRadius': '5px',
+                                'textAlign': 'center',
+                                'margin': '10px'
+                            },
+                        ),
+                        html.Button(
+                            id='run-btn',
+                            className='app__button',
+                            children='Run'
+                        )
+                    ]
+                )
+            ]
+        ),
+        html.Div(
+            id='bottom-panel-div',
+            className='app__container',
+            children=[
+                html.Div(
+                    id='parameters-div',
+                    className='app__parameters',
+                    title='Parameters description',
+                    children=[
+                        html.Div(
+                            id='param-tech-div',
+                            className='app__parameters__section',
+                            children=[
+                                html.Div('Technische Parameter'),
+                                html_param_input('Zyklenwirkungsgrad', 76.6, '%'),
+                                html_param_input('Entladetiefe', 37, '%'),
+                                html_param_input('C-Rate', 0.5),
+                            ]
+                        ),
+                        html.Div('Investitionskosten-div'),
+                        html.Div('Betriebskosten-div'),
+                    ]
+                ),
+                html.Div(
+                    id='results-div',
+                    className='app__results',
+                    title='Results description',
+                    children=[
+                        html.Div('Ergebnisse'),
+                        dcc.Dropdown(
+                            id='country-input',
+                            className='app__input__dropdown__map',
+                            options=[],
+                            value=None,
+                            multi=False
+                        )
+                    ]
+                ),
+            ]
+
+        )
+    ],
+)
+
+
+@app.callback(
+    Output('data-store', 'data'),
```

Comment:
the callbacks allow interactions between the components based on the `id` of each component